todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksLocalDataSource.java

```diff
@@ -118,12 +119,28 @@ public static void destroyInstance() {
     @Override
     public void saveTask(@NonNull Task task) {
         checkNotNull(task);
+        ContentValues values = toContentValues(task);
+        mDatabaseHelper.insert(TaskEntry.TABLE_NAME, values, SQLiteDatabase.CONFLICT_REPLACE);
+    }
+
+    @Override
+    public Completable saveTasks(@NonNull List<Task> tasks) {
+        return Observable.using(mDatabaseHelper::newTransaction,
+                transaction -> Observable.from(tasks)
```

Comment:
could you extract `Observable.from .... .toList()` to another method, to make things a bit more readable?